joomla/joomla-cms
# [4.0] Start converting actionlogs to new format

### Summary of Changes
This will be a conversion of the com_actionlogs component to use the new Joomla 4 structure.

### Testing Instructions
#### Component Action Logs
1. Go to Users -> Users Action Log
2. Check that the table of actions show
3. Check that the search works
4. Check that the filtering works
5. Check that the Options work

#### User Editing
1. Go to Users -> Manage
2. Edit an admin user
3. Check that the User Actions Log Options tab shows
4. Check that the options are saved

#### Latest Action Logs Module
1. Go to Content -> Administrator Modules
2. Check that the Latest Actions module is enabled. If not, enable the module
3. Go to Control Panel
4. Check that the module shows

#### Plugins
1. Go to System -> Plugins
2. Edit the `System - User Actions Log` and check that the settings can be saved
3. Edit the `Action Log - Joomla` and check that the settings can be saved
4. Edit the `Privacy - Action Logs` and check that the settings can be saved

Enable the plugins and check that the functionality still works.

Use Action Logs as usual and make sure it all works as expected.

### Expected result
It works as before the changes



### Documentation Changes Required
None
